### DISCUSSION TITLE
Invisible occlusion mesh !

### DISCUSSION THREAD

**Discussion by Bettole** [google/model-viewer#4793] [Q&A]
### Discussed in https://github.com/google/model-viewer/discussions/2797

<div type='discussions-op-text'>

<sup>Originally posted by **dthery** September 16, 2021</sup>
Can we use the colorWrite property of three.js to create an invisible occlusion mesh that hides some areas of a model to make transparency? 
https://threejs.org/docs/#api/en/materials/Material.colorWrite</div>

Please Help !

**Comment by elalish** [comment]
That's outside of our public API - feel free to fork us and hack to your heart's content!
